test/framework/gardener_utils.go

```diff
@@ -666,3 +667,75 @@ func (f *GardenerFramework) WaitForSeedToBeDeleted(ctx context.Context, seed *ga
 		return retry.MinorError(fmt.Errorf("seed %q still exists", seed.Name))
 	})
 }
+
+// CreateManagedSeed creates a new managed seed and waits for it to be created and successfully reconciled.
+func (f *GardenerFramework) CreateManagedSeed(ctx context.Context, managedSeed *seedmanagementv1alpha1.ManagedSeed) error {
+	// Create the managed seed
+	err := retry.UntilTimeout(ctx, 20*time.Second, 5*time.Minute, func(ctx context.Context) (done bool, err error) {
+		err = f.GardenClient.DirectClient().Create(ctx, managedSeed)
```

Comment:
`Create/Update/Delete/Patch` always go directly to the API server, so no need to use the `DirectClient` for those calls.
```suggestion
		err = f.GardenClient.Client().Create(ctx, managedSeed)
```
Note to self: I really need to clean this up and make it a `APIReader` instead. Will hopefully start with this next week as part of https://github.com/gardener/gardener/issues/2822.

Reply:
Done